tests/clickhouse-test

```diff
@@ -111,12 +111,21 @@ def clickhouse_execute_http(
     if default_format is not None:
         params["default_format"] = default_format
 
-    client.request(
-        "POST",
-        f"/?{base_args.client_options_query_str}{urllib.parse.urlencode(params)}",
-    )
-    res = client.getresponse()
-    data = res.read()
+    for i in range(MAX_RETRIES):
+        try:
+            client.request(
+                "POST",
+                f"/?{base_args.client_options_query_str}{urllib.parse.urlencode(params)}",
+            )
+            res = client.getresponse()
+            data = res.read()
+            break
+        except ex:
```

Comment:
I guess it is better to enumerate exceptions here to retry on.

And by the way, what problem does it fixes?

Reply:
Just saw multiple `Connection reset by peer` for some random reason. And I don't think we need to check some special exceptions because any HTTP code is not an exception, only connection loss.

Reply:
It can be some 5xx, i.e. `TABLE_ALREADY_EXISTS` and similar